opencti-platform/opencti-graphql/src/parser/csv-mapper.ts

```diff
@@ -135,7 +135,7 @@ const handleDirectAttribute = (attributeKey: string, column: AttributeColumn, in
     throw UnsupportedError('Invalid attribute', { key: attributeKey, type: entity_type });
   }
   const computedValue = computeValue(value, column, attributeDef);
-  if (computedValue) {
+  if (computedValue !== null && computedValue !== undefined) {
```

Comment:
question: from the test below I see that when there is an invalid value (empty) the line is dropped. Don't we need to throw an exception to warn the end user that the line has been dropped when computerValue is null or undefined (line 'Gerard' in malware.csv) ?

Reply:
To discuss with product team. For what I understood, giving files with missing data is very common so I don't know if it's relevant or not. In any case, if to be done, should be in a specific feature

Reply:
ok thanks

Reply:
it's already the case I guess, since there is a test in computeValue which will return null if the value is empty, which might be an issue. If you have an empty value (empty string or empty object or empty array), it will ignore the line ?

Reply:
The line is not always ignored.
If a cell is empty, no matter the type (string, number, whatever), then we set null for the associated property of the entity.
Then, if this property is required in the schema, then the line is dropped, otherwise we keep it, just some properties are not filled but not an issue as they are not required